Makefile

```diff
@@ -1,8 +1,14 @@
-.PHONY: help test clean dockerbuild dockerserve dockershell dockerprod geoip
+.PHONY: help test clean dockerbuild dockerserve dockershell dockerprod geoip ipproxy
+
 
-GEOIP_UPDATE = geoipupdate
 GEOIP_DIR = geoip
-GEOIP_CONF_FILE = $(GEOIP_DIR)/GeoIP.conf
+GEOIP_CITY_FILE = dbip-city-lite-2020-07.mmdb.gz
+GEOIP_COUNTRY_FILE = dbip-country-lite-2020-07.mmdb.gz
+GEOIP_CITY_DB_URL = https://download.db-ip.com/free/dbip-city-lite-2020-07.mmdb.gz
+GEOIP_COUNTRY_DB_URL = https://download.db-ip.com/free/dbip-country-lite-2020-07.mmdb.gz
+
+TOR_EXIT_LIST_FILE = torbulkexitlist
+TOR_EXIT_LIST_URL = https://check.torproject.org/torbulkexitlist
```

Comment:
Same here.

Reply:
I actually want these to be different so the downloadable name can be different from the URL path. The local file name needs to be under our control while the remote file isn't under our control.